docs/applications/development/install-r-run-r-code-through-java-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,242 @@
+---
+author:
+    name: Shilong Zhuang
+    email: huojianxixi@gmail.com
+description: 'Set up and configure R software at your linode Server(i.e. Ubuntu 14.04) to run Java applications to execute R code'
+keywords: 'R,rjava,forecast'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Sunday, December 27th, 2015'
+modified: Monday, December 28th, 2015
+modified_by:
+    name: Shilong Zhuang
+title: 'How to install R and run R code through Java on Ubuntu 14.04'
+contributor:
+    name: Shilong Zhuang
+    link: https://github.com/huojianxixi
+external_resources:
+ - '[Talking R through Java](http://www.r-bloggers.com/talking-r-through-java/)'
+---
+
+*This is a Linode Community guide. [Write for us](/docs/contribute) and earn $250 per published guide.*
+
+<hr>
+
+This guide walks through you how to install R on Ubuntu 14.04 by linode server and run the Java code to execute the R code with a forecast example. [R](https://www.r-project.org/about.html) is a free software environment for statistical computing and graphics. If you are R fans and want to know how to use Java to execute R code on Ubuntu linux server, then try to read this guide you would know:
+
+1. How to install R in linode linux server
+2. How to configure Java for R code in linux server
+3. An example to run forecast R code through Java in the linux server
+
+## Before You Begin
+
+1. Make sure you login with a **sudo user**, otherwise you should use **root** in case. 
+
+2. Ensure you are already have installed Java, try `java -version` and `javac -version`, you can find the below result for sure.
+
+        root@localhost:/# java -version
+        java version "1.8.0_45"
+        Java(TM) SE Runtime Environment (build 1.8.0_45-b14)
+        Java HotSpot(TM) 64-Bit Server VM (build 25.45-b02, mixed mode)
+        root@localhost:/# javac -version
+        javac 1.8.0_45
+
+3. Update your system:
+
+        sudo apt-get update
+
+{: .note}
+>
+> The above that mentioned `1.8.0_45` may be different from your Java version.
+
+
+## Install the R
+
+1. Enter the following command at the terminal

```

Comment:
Rather than tell the reader to execute a command, tell them what action they are performing.